net-analyzer/ipsumdump/ipsumdump-1.85.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Simple TCP/IP Dump summarizer/analyzer"
+HOMEPAGE="http://read.seas.harvard.edu/~kohler/ipsumdump/"
+SRC_URI="http://read.seas.harvard.edu/~kohler/ipsumdump/ipsumdump-1.85.tar.gz"
+
+LICENSE="the-Click-license"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+ipv6 +nanotimestamp"
+
+DEPEND="dev-lang/perl
+	dev-libs/expat
+	net-libs/libpcap
+	sys-apps/texinfo"
+RDEPEND="${DEPEND}"
+
+WANT_AUTOCONF=2.60
```

Comment:
``WANT_AUTOCONF`` requires ``autotools.eclass`` to do anything (and IIRC it needs to be defined before it's inherited).